lib/middleware/locals.coffee

```diff
@@ -49,6 +49,5 @@ module.exports = (req, res, next) ->
     (ua.match(/BB10/i)) ||
     (ua.match(/BlackBerry/i))
   )
-  res.locals.sd.HIGHLIGHT_HEADER_SEARCH = req.query?.highlight_header_search
```

Comment:
Ah, since it's called the same thing `HIGHLIGHT_HEADER_SEARCH=highlighted` you don't have to make an update to the place that checks this? Slick.